static/app/utils/replays/hooks/useDeadRageSelectors.tsx

```diff
@@ -6,24 +6,33 @@ import {
   DeadRageSelectorQueryParams,
 } from 'sentry/views/replays/types';
 
-export default function useRageDeadSelectors(
-  params: DeadRageSelectorQueryParams = {per_page: 10, sort: '-count_dead_clicks'}
-) {
+export default function useRageDeadSelectors(params: DeadRageSelectorQueryParams) {
   const organization = useOrganization();
   const location = useLocation();
   const {query} = location;
 
-  return useApiQuery<DeadRageSelectorListResponse>(
-    [
-      `/organizations/${organization.slug}/replay-selectors/`,
-      {
-        query: {
-          ...query,
-          per_page: params.per_page,
-          sort: params.sort,
+  const {isLoading, isError, data, getResponseHeader} =
+    useApiQuery<DeadRageSelectorListResponse>(
+      [
+        `/organizations/${organization.slug}/replay-selectors/`,
+        {
+          query: {
+            cursor: query.cursor,
+            environment: query.environment,
+            project: query.project,
+            statsPeriod: query.statsPeriod,
+            per_page: params.per_page ?? 10,
```

Comment:
10 is the default from the backend! wasn't sure if it was better to be explicit or not. can take out the `?? 10` if it's not necessary

Reply:
Ah, I would allow backend to handle the default (otherwise we would have 2 places where we need to keep it in sync), unless there's a specific reason to make default 10 here.